architecture/04-canvas-nodes.md

```
@@ -0,0 +1,91 @@
+  actorNode: ActorNode,   // ← new
+  systemNode: SystemNode, // ← new
+};
+```
+
+---
+
+## Node Types
+
+### `actorNode` — Human actor (User, Admin, etc.)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)